src/porepy/geometry/domain.py

```diff
@@ -177,6 +177,22 @@ def __contains__(self, other: object) -> bool:
                         return False
                 return True
 
+    def side_lengths(self) -> np.ndarray:
+        """Compute the lengths of the bounding box sides.
+
+        Returns:
+            Lengths of the sides of the bounding box.
+
+        """
+
+        x_length = self.bounding_box["xmax"] - self.bounding_box["xmin"]
+        y_length = self.bounding_box["ymax"] - self.bounding_box["ymin"]
+        if self.dim == 2:
```

Comment:
Although 1d simulations in principle should be possible, it is sufficiently safe to not take this into account here, I assume? Because, who will run 1d PP simulations?

Reply:
I simply copied the logic from the rest of this file.